kratos/python/add_constraint_to_python.cpp

```diff
@@ -0,0 +1,47 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala
+//
+
+
+// System includes
+
+// External includes
+
+
+// Project includes
+#include "includes/define_python.h"
+#include "add_constraint_to_python.h"
+#include "utilities/constrain.h"
+
+namespace Kratos
+{
+namespace Python
+{
+
+using namespace pybind11;
+
+void  AddConstraintToPython(pybind11::module& m)
+{
+
+    class_<MasterSlaveConstraint>(m, "MasterSlaveConstraint" )
+	.def(init<>())
```

Comment:
i think we should expose a richer constructor

Reply:
i mean somethign like

     MasterSlaveConstraint( 
          master_dofs. 
          slave_dofs,
          T,
          b
       )